frontends/election-manager/src/screens/write_ins_screen.tsx

```diff
@@ -27,8 +27,13 @@ const ContentWrapper = styled.div`
   }
 `;
 
+const ResultsFinalizedNotice = styled.p`
+  color: rgb(71, 167, 75);
```

Comment:
Where did this color come from? I'm wondering if we have any colors defined or other patterns to follow already that we could use here.

Reply:
Actually meant to ask if we have any centrally defined colors I might be able to reference - copied this one from the "primary" color on our button component, but would prefer a named variable or something if we have one

Reply:
Yeah this might be a good q for @beausmith